frontend/src/app/main/ui/workspace/plugins.cljs

```diff
@@ -80,7 +80,7 @@
    ::mf/register-as :plugin-management}
   []
 
-  (let [plugins-state* (mf/use-state @plugins/pluginsdb)
+  (let [plugins-state* (mf/use-state (preg/plugins-list))
```

Comment:
`(preg/plugins-list)` should be probably `#(preg/plugins-list)` for avoid calling it on each render

Reply:
Changed